lib/src/fixed.dart

```diff
@@ -543,19 +542,31 @@ class Fixed implements Comparable<Fixed> {
   static Fixed? tryParse(
     String amount, {
     int scale = 2,
-    bool invertSeparator = false,
+    String decimalSeparator = '.',
+    String groupSeparator = ',',
   }) {
     try {
-      return Fixed.parse(amount,
-          //pattern: pattern,
-          scale: scale,
-          invertSeparator: invertSeparator);
+      return Fixed.parse(
+        amount,
+        //pattern: pattern,
+        scale: scale,
+        decimalSeparator: decimalSeparator,
+        groupSeparator: groupSeparator,
+      );
       // ignore: avoid_catches_without_on_clauses
     } catch (_) {
       return null;
     }
   }
 
+  /// Returns a JSON representation of the [Fixed] value.
+  ///
+  /// The JSON representation includes the [minorUnits] and [scale].
+  Map<String, dynamic> toJson() => {
+        'minorUnits': minorUnits.toString(),
+        'scale': scale,
```

Comment:
change to decimalDigits - this a fairly significant breaking changes but provides a long term advance.